eng/NullablePolyfill.targets

```diff
@@ -0,0 +1,28 @@
+<Project>
+
+  <Choose>
+    <!--
+        C# 8.0's nullable annotations only work for .NET Core 3.0+ by default. However it can be made to work for other target frameworks
+        by compiling the annotations (as internal types).
+        The code block that follows sets it up so projects in this repo that target ns2.0 or netfx can compile when Nullable is configured.
+        Based on https://github.com/dotnet/runtime/blob/4d37f40c49ed7f03d557e96323835bb87638b6ad/src/libraries/Directory.Build.targets#L171-L175
+
+        SuppressNullableAttributesImport to disable. Used when attributes are already present from another project because of InternalsVisibleTo.
+    -->
+    <When Condition="'$(Nullable)' != '' and
+                     '$(Nullable)' != 'disable' and
+                     '$(SkipIncludeNullableAttributes)' != 'true' and
+                     '$(MSBuildProjectExtension)' == '.csproj' and
+                     '$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
+      <PropertyGroup>
+        <DefineConstants>$(DefineConstants),INTERNAL_NULLABLE_ATTRIBUTES</DefineConstants>
```

Comment:
We never want to make these attributes public. I would just get rid of this DefineConstant and delete all the `public` modifiers in `NullableAttributes.cs`.